src/mod_bosh_socket.erl

```diff
@@ -251,15 +254,18 @@ closing(Event, State) ->
 %% @end
 %%--------------------------------------------------------------------
 accumulate(Event, _From, State) ->
-    ?DEBUG("Unhandled sync event in 'accumulate' state: ~w~n", [Event]),
+    ?LOG_DEBUG(#{what => bosh_socket_unhandled_sync_event, state => accumulate,
+        event => Event}),
     {reply, ok, accumulate, State}.
 
 normal(Event, _From, State) ->
-    ?DEBUG("Unhandled sync event in 'normal' state: ~w~n", [Event]),
+    ?LOG_DEBUG(#{what => bosh_socket_unhandled_sync_event, state => normal,
```

Comment:
`state_name` instead of `state`.
And indention for record should be different (second line should start below what).